### ISSUE TITLE
Open menu bar item on mouse down rather than mouse up

### ISSUE THREAD

**Issue by WhiteMac** [p0deje/Maccy#299]
Hi,

I mostly use Maccy via the menu bar icon. Most menu bar items and right-click menus on Mac open on mouse down, rather than mouse up. This allows for a great shortcut of effectively using menus with a single click. You hold the mouse down to open the menu, hover over the option you want, and release the mouse to select the option. Maccy, however, currently requires two clicks. One to open the menu, another to select the option. It'd be great if Maccy opened on mouse down like most menus and saved a click.

Thank you.

**Comment by p0deje** [comment]
You can fix this by enabling "Avoid taking application focus" in advanced preferences.

**Comment by WhiteMac** [comment]
Hello, it was already enabled, and I tried disabling and re-enabling it, but it hasn't fixed the issue. Maccy menu bar item opens on mouse up for me, not on mouse down.

**Comment by p0deje** [comment]
You are right, I've been testing the newer version where this issue is already fixed. It will work fine in the next released version. Stay tuned!

**Comment by momasta** [comment]
Hello @p0deje, Maccy no longer opens on mouseDown in version 2.3.0 (54).
When I press and hold the menu bar icon, nothing happens until I release the mouse button.
("Avoid taking application focus" is gone from advanced preferences.)

Instead, I'd expect it to work just like macOS' Wi-Fi icon:
- Click and hold the icon,
- Hover over the menu while keeping the mouse button held,
- Release on the desired item to activate it.
Just like the author of this issue described it.

Can you take a look a it, please? 

Thank you for this great app by the way.
